src/confluent_kafka/src/Producer.c

```diff
@@ -814,6 +891,13 @@ static PyMethodDef Producer_methods[] = {
         { "set_sasl_credentials", (PyCFunction)set_sasl_credentials, METH_VARARGS|METH_KEYWORDS,
            set_sasl_credentials_doc
         },
+        { "topic_new", (PyCFunction)Producer_topic_new, METH_VARARGS|METH_KEYWORDS,
+          "Creates a new topic handle for topic named topic, allows setting a per-topic configuration\n"
+          "\n"
+          "  :param str topic: Topic to create\n"
+          "  :param dict conf: Configuration properties\n"
```

Comment:
Can this even set the servers? or those settings are limited to the producer config?